src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.AggregatingTagger.cs

```diff
@@ -53,6 +53,7 @@ public void Dispose()
                 foreach (var kvp in _idToProviderAndTagger)
                 {
                     var tagger = kvp.Value.Item2;
+
```

Comment:
Was refactoring this a bit.  Then decided to revert that part.  Guess a newline snuck in.  :)